operator/cmd/mesh/testdata/manifest-generate/output/all_on.golden-show-in-gh-pull-request.yaml

```diff
@@ -624,7 +624,7 @@ spec:
             - --proxyAdminPort
             - "15000"
             - --controlPlaneAuthPolicy
-            - NONE
+            - MUTUAL_TLS
```

Comment:
this does not look correct, it should be NONE. we probably need to remove this flag longer term